configs/partials/webpack.config.base.js

```diff
@@ -12,6 +12,19 @@ const CheckTargetNet = require('../../internals/scripts/CheckTargetNet');
 
 CheckTargetNet();
 
+const targetNet = process.env.TARGET_NET;
+
+let userData = 'Concordium Wallet';
+if (targetNet) {
+    userData += ` ${targetNet}`;
+}
+
+if (targetNet && !['stagenet', 'testnet'].includes(targetNet)) {
```

Comment:
Why was this check added? We already have `CheckTargetNet()` above.

Reply:
Some sloppy merging from my side.
Though it did make me notice another error. :)